geomstats/geometry/discretized_curves.py

```diff
@@ -15,25 +15,36 @@
 
 
 class DiscretizedCurves(Manifold):
-    """Space of discretized curves sampled at points in ambient_manifold."""
+    """Space of discretized curves sampled at points in ambient_manifold.
+
+    Parameters
+    ----------
+    ambiant_manifold : Manifold
+        Manifold in which curves take values.
+    """
 
     def __init__(self, ambient_manifold):
```

Comment:
Why not having n_landmarks as a parameter of the constructor, like for the class `Landmarks`?
Or having `lambda n : L2Metric(manifold, n)` for both?